src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -151,6 +156,8 @@ const formConfig = {
         'Your saved disability compensation application (21-526EZ) has expired. If you want to apply for disability compensation, please start a new application.',
       saved: 'Your disability compensation application has been saved.',
     },
+    // optional, redirect is in Form5256EZApp
+    // restartFormCallback: () => '/',
```

Comment:
intentional?

Reply:
It is intentionally commented out... it was left in there as sort of documentation, but now I've created a document PR. I'd remove it now, but I really don't want to wait 30+ minutes to rebuild this PR just to remove a comment :D